src/index.html

```diff
@@ -7,8 +7,36 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+      >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home">
+        <img
+        class="logo"
+        src="images/logo.png"
+        alt="moyo-logo"
+        >
+      </a>
+      <nav class="nav">
+        <a class="nav__list is-active" href="#apple">Apple</a>
+        <a class="nav__list" href="#samsung">Samsung</a>
+        <a class="nav__list" href="#smarts-and-phones">Смартфоны и телефоны</a>
+        <a
+          data-qa="hover"
+          class="nav__list"
+          href="#laps-and-PC">Ноутбуки и компьютеры
+        </a>
+        <a class="nav__list" href="#gadgets">Гаджеты</a>
+        <a class="nav__list" href="#tablets">Планшеты</a>
+        <a class="nav__list" href="#photos">Фото</a>
+        <a class="nav__list" href="#videos">Видео</a>
```

Comment:
<img width="400" alt="image" src="https://user-images.githubusercontent.com/90685701/167106298-c80d54ef-e120-4a17-ae7b-1870e105073d.png">
you should probably use some kind of list here